build/Helix/PrepareHelixPayload.ps1

```diff
@@ -26,10 +26,18 @@ cp build/Helix/ConvertWttLogToXUnit.cs $payloadDir
 
 cp build/Helix/ConvertWttLogToXUnit.ps1 $payloadDir
 
-cp $Platform-$Configuration-taeftest/ProjectReunionHelix.dll $payloadDir
+cp BuildOutput/$Configuration/$Platform/BaseUnitTests/BaseUnitTests.dll $payloadDir
 
-cp build/Helix/packages/Taef.Redist.Wlk.10.57.200918006-develop/build/Binaries/$Platform/* $payloadDir
+#cp BuildOutput/$Configuration/$Platform/UnitTest/UnitTest.dll $payloadDir
 
-cp build/Helix/packages/Taef.Redist.Wlk.10.57.200918006-develop/build/Binaries/$Platform/CoreClr/* $payloadDir
+cp dev/MRTCore/mrt/packages/Taef.Redist.Wlk.10.57.200918006-develop/build/Binaries/$Platform/* $payloadDir
```

Comment:
>Taef.Redist.Wlk.10.57.200918006- [](start = 28, length = 32)

this is fragile

Reply:
Agreed, but at least it is easy to find by whoever is updating the dependency feed ...

Reply:
I agree, I updated my comment above with how WinUI3 handles this and I think it is probably a good solution we can update this going forward instead.